crates/common/tedge_config/Cargo.toml

```diff
@@ -8,9 +8,10 @@ rust-version = "1.58.1"
 [dependencies]
 certificate = { path = "../certificate" }
 serde = { version = "1.0", features = ["derive"] }
-tedge_utils = { path = "../tedge_utils" }
+tedge_utils = { path = "../tedge_utils", features = ["tedge-derive"] }
 tempfile = "3.2"
 thiserror = "1.0"
+tracing = "0.1"
```

Comment:
This is just to emit warnings when some config parameters are unknown? Cannot this be done from the caller and not implicitly on reading a config?